### ISSUE TITLE
False negative: 'label' rule should fail if a <select> element has an empty implicit label

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#754]
Example:
```
<label>
    <span class="label"></span>			
    <select>
        <option value="1" selected="selected">Please choose a region</option>
        <option value="2">Coastal</option>
        <option value="3">Forest</option>
        <option value="4">Grasslands</option>
        <option value="5">Mountains</option>
    </select>
</label>
```

**Comment by dylanb** [comment]
@iamrafan This is definitely an issue. Is it ok if we fix this in the release after 3.0.0?

**Comment by iamrafan** [comment]
@dylanb Sure

**Comment by marcysutton** [comment]
The accessibleTextVirtual function is counting the select's text as the label–"Please choose a region" is announced in Voiceover, and likely other ATs. But that doesn't help if the first option is "Desert" instead. Working on a fix.
